skimage/metrics/__init__.py

```diff
@@ -12,5 +13,6 @@
            'mean_squared_error',
            'normalized_root_mse',
            'peak_signal_noise_ratio',
-           'structural_similarity'
+           'structural_similarity',
+           'normalized_mutual_information',
```

Comment:
Mutual information would be worth a gallery example for itself I think. This PR is already enormous, so can you open an issue to remind us that we'll need to add an example once it is merged?

Reply:
Created: https://github.com/scikit-image/scikit-image/issues/6427